docker-compose.yml

```diff
@@ -86,3 +96,5 @@ services:
       - SQL_SERVER_PORT=1433
       - CASSANDRA_HOST=cassandra
       - CASSANDRA_PORT=9042
+      - ORIENTDB_HOST=orientdb
+      - ORIENTDB_PORT=2424
```

Comment:
Actually these two lines can be removed because these values are used only in one place.